### PR TITLE
test_util: use real uuid in mocktikv.

### PR DESCRIPTION
**Pull request by winkyao**
```
time="2018-03-28T17:31:10+08:00" level=warning msg="[0] compile error:
[schema:1146]Table 'mysql.global_variables' doesn't exist
select HIGH_PRIORITY * from mysql.global_variables where variable_name in ('autocommit', 'sql_mode', 'max_allowed_packet', 'time_zone', 'tidb_skip_utf8_check', 'tidb_index_join_batch_size', 'tidb_index_lookup_size', 'tidb_index_lookup_concurrency', 'tidb_index_serial_scan_concurrency', 'tidb_distsql_scan_concurrency')"
time="2018-03-28T17:31:10+08:00" level=error msg="Failed to load common global variables."
time="2018-03-28T17:31:10+08:00" level=info msg="stop DDL:eabf22c3-53a9-4bba-97d2-ba15dfd38e57"
panic: close of closed channel

goroutine 14536 [running]:
github.com/pingcap/tidb/domain.(*Domain).Close(0xc48b49e5a0)
        /home/pingcap/go/src/github.com/pingcap/tidb/domain/domain.go:389 +0x44
github.com/pingcap/tidb/session.runInBootstrapSession(0x1ca2860, 0xc453d59b20, 0x144c500)
        /home/pingcap/go/src/github.com/pingcap/tidb/session/session.go:1150 +0x149
github.com/pingcap/tidb/session.BootstrapSession(0x1ca2860, 0xc453d59b20, 0x0, 0x1ca2860, 0xc453d59b20)
        /home/pingcap/go/src/github.com/pingcap/tidb/session/session.go:1092 +0x33c
github.com/pingcap/tidb/util/kvencoder.newMockTikvWithBootstrap(0xc400000020, 0x144c6c0, 0xc43aac3e68, 0xc43aac3dd0, 0xc43aac3dc8)
        /home/pingcap/go/src/github.com/pingcap/tidb/util/kvencoder/kv_encoder.go:201 +0xc6
github.com/pingcap/tidb/util/kvencoder.(*kvEncoder).initial(0xc48f3d9f80, 0xc420314e30, 0xc, 0x1c9d9c0, 0xc48f4746d0, 0x0, 0x0)
        /home/pingcap/go/src/github.com/pingcap/tidb/util/kvencoder/kv_encoder.go:232 +0xed
github.com/pingcap/tidb/util/kvencoder.New(0xc420314e30, 0xc, 0x1c9d9c0, 0xc48f4746d0, 0x0, 0x0, 0x184, 0x19e)
        /home/pingcap/go/src/github.com/pingcap/tidb/util/kvencoder/kv_encoder.go:90 +0x6c
github.com/pingcap/tidb-lightning/ingest/kv.NewTableKVEncoder(0x0, 0x0, 0xc420314e30, 0xc, 0x214, 0xc420477478, 0x8, 0x21a, 0xb, 0xc460be3380, ...)
        /home/pingcap/go/src/github.com/pingcap/tidb-lightning/ingest/kv/sql2kv.go:49 +0xa1
github.com/pingcap/tidb-lightning/ingest/restore.(*kvEncoderPool).init.func1(0xc4922450b0, 0xc4922c2880)
        /home/pingcap/go/src/github.com/pingcap/tidb-lightning/ingest/restore/restore.go:358 +0xec
created by github.com/pingcap/tidb-lightning/ingest/restore.(*kvEncoderPool).init
        /home/pingcap/go/src/github.com/pingcap/tidb-lightning/ingest/restore/restore.go:356 +0x140
```
Panic in lightning, because the mocktikv use an id that may be duplicate to create the store, so the domain maybe closes twice unexpectedly.

### CODE REVIEW

**Comment by winkyao** [comment]
@huachaohuang @coocood @tiancaiamao PTAL

**Comment by huachaohuang** [comment]
LGTM

**Comment by jackysp** [comment]
Do we need a test case for it?

File: store/tikv/test_util.go [hunk 1]
```diff
@@ -14,13 +14,10 @@
...
 
 import (
-	"fmt"
-	"math/rand"
-	"time"
-
 	"github.com/juju/errors"
 	"github.com/pingcap/pd/pd-client"
 	"github.com/pingcap/tidb/kv"
+	"github.com/twinj/uuid"
```
Comment (tiancaiamao) [inline store/tikv/test_util.go]: vendor?

**Comment by tiancaiamao** [comment]
I think this error is unlikely to be caused by random ID conflict.

File: store/tikv/test_util.go (hunk 1 above)
Comment (coocood) [reply store/tikv/test_util.go]: already vendorred

**Comment by winkyao** [comment]
@tiancaiamao Use uuid in mocktikv has no side effect.

**Pull request merged**